app/assets/javascripts/discourse/app/components/invite-panel.js

```diff
@@ -26,8 +27,6 @@ export default Component.extend({
   inviteIcon: "envelope",
   invitingExistingUserToTopic: false,
 
-  isAdmin: alias("currentUser.admin"),
```

Comment:
This removal seem to be missing a matching addition of `isAdmin: readOnly(…)`?

Reply:
Good catch, fixed.